cypress/e2e/homepage.cy.js

```diff
@@ -18,8 +18,47 @@ describe("the homepage", () => {
   });
 
   it("2048 Expert link redirects to its web page", () => {
-    cy.get('a[id="2048expert-btn"]').should("contain.text", "2048 Expert");
-    cy.get('a[id="2048expert-btn"]').click()
+    cy.get('a[id="a2048expert-btn"]').should("contain.text", "2048 Expert");

```

Comment:
Just wondering, why `2048-expert-btn` has been changed to `a2048-expert-btn`?

Reply:
iirc css didn't like the number at the start of the id so I changed it to `a2048expert-btn`